gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaSource.java

```diff
@@ -526,37 +529,31 @@ public boolean apply(KafkaTopic input) {
       return this.topicNamePattern.matcher(input.getName()).matches();
     }
   }
-  
-  private class WorkUnitCreator implements Runnable{
+
+  private class WorkUnitCreator implements Runnable {
     private final KafkaTopic topic;
     private final SourceState state;
-    private final Optional <State> topicSpecificState;
+    private final Optional<State> topicSpecificState;
     private final Map<String, List<WorkUnit>> allTopicWorkUnits;
-    
-    WorkUnitCreator(KafkaTopic topic, SourceState state, Optional<State> topicSpecificState, 
-        Map<String, List<WorkUnit>> workUnits){
+
+    WorkUnitCreator(KafkaTopic topic, SourceState state, Optional<State> topicSpecificState,
+        Map<String, List<WorkUnit>> workUnits) {
       this.topic = topic;
       this.state = state;
       this.topicSpecificState = topicSpecificState;
       this.allTopicWorkUnits = workUnits;
     }
-    
+
     @Override
     public void run() {
       try {
-        long curTime = System.currentTimeMillis();
-        LOG.info("Using thread pool to create work units for topic " + this.topic.getName());
-        List<WorkUnit> oneTopicWorkUnits =
-            KafkaSource.this.getWorkUnitsForTopic(this.topic, this.state, topicSpecificState);
-        this.allTopicWorkUnits.put(this.topic.getName(), oneTopicWorkUnits);
-        LOG.info("Done creating work unit for " + this.topic.getName() + ", created work unit size is "
-            + oneTopicWorkUnits.size() + ". Total time used for creating this topic is "
-            + (System.currentTimeMillis() - curTime) + " milliseconds");
+        this.allTopicWorkUnits.put(this.topic.getName(),
+            KafkaSource.this.getWorkUnitsForTopic(this.topic, this.state, topicSpecificState));
```

Comment:
The log is removed here. There will be log for creating workunits for each partition, so these two log messages are not really useful.